### PR TITLE
Build docker image and push to Docker Hub

### PR DESCRIPTION
**Pull request by dlabordus**

### CODE REVIEW

**Review by Flurb — CHANGES_REQUESTED** [review]

File: .github/workflows/npm_build.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: NodeJS Build
+
+on:
+  push:
```
Comment (Flurb) [inline .github/workflows/npm_build.yml:4]: on: push, make it consistent with other Github Actions :)

**Review by Flurb — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerhub_publish.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - name: Build and push image to Docker Hub
+        uses: docker/build-push-action@v2
+        with:
+          # Set the context to use the current directory and not execute it's own git checkout.
+          context: .
+          push: true
+          # Tag the images using the tagname and also latest.
+          tags: |
+            lfenergycompas/compas-open-scd:${{ steps.extract_tagname.outputs.tagname }}
+            lfenergycompas/compas-open-scd:latest
```
[the remark this answers is not in the document]
Comment (dlabordus) [reply .github/workflows/dockerhub_publish.yml:45]: latest should also point to the latest stable released version.

File: .github/workflows/npm_build.yml (hunk 1 above)
Comment (dlabordus) [reply .github/workflows/npm_build.yml]: Fixed.

File: .github/workflows/dockerhub_publish.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dlabordus) [reply .github/workflows/dockerhub_publish.yml:45]: During NPM Build I also added building the image, but not pushing it to DockerHub.
This way the docker build is also tested.

**Review by Flurb — APPROVED** [review]

**Pull request merged**